src/common/options/global.yaml.in

```diff
@@ -5516,6 +5516,19 @@ options:
   - hdd
   - ssd
   with_legacy: true
+- name: bluestore_allocator_lookup_policy
+  type: str
+  level: advanced
+  desc: Determines how to perform the next free extent lookup.
+  long_desc: With '''hdd_optimized'' allocator uses the previous found position to start the next lookup from.
+    This could help in making writes to disk contiguous. ''ssd-optimized'' instructs allocator to perform lookup
+    starting from volume's beginning. This could help SSD disks firmware to perform garbage collection more
+    efficiently. With ''auto'' policy bluestore uses the mode corresponding to the actual device type.
```

Comment:
Line 5523 has an extra quote.

Suggest:

When set to 'hdd_optimized' the allocator searches from the last location found.  This may facilitate contiguous disk writes.  It may similarly be beneficial for large-IU QLC SSDs to enable firmware coalescing of sub-IU writes.  When set to 'ssd-optimized' the allocator will search from the beginning of the device.  This may facilitate SSD firmware housekeeping.  When set to 'auto' the value will be derived from the detected device type (rotational or non-rotational).

Reply:
Note to self:  after merge, improve wording in adjacent entries.